server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/tservers.js

```diff
@@ -18,29 +18,25 @@
  */
 "use strict";
 
-var tserversList;
+var tserversTable;
 /**
  * Creates tservers initial table
  */
 $(document).ready(function() {
-    
+
+  // hide the note about highlighted rows by default
+  $('#recovery-caption').hide();
```

Comment:
Should be able to hide this on the FTL file. If you do it here, every time the page loads, you may see it flash on the screen. The Manager page (manager.ftl) has an alert that is hidden by default using `style="display: none;"`

Reply:
I'll try that out but I was under the impression that it needs to happen here because this code will run again when the table is refreshed which will clear the caption if needed. I think if it were in the FTL it might never be hidden again once it is shown.

Reply:
This is how the tables are handled too. On refresh you can see the non functioning servers table and the dead servers table flash on the screen before going away.

Reply:
You can avoid the flash by having the initial style be hidden, and then `.show()` and `.hide()` it later, as circumstances require.

Reply:
addressed in [05e9e7a](https://github.com/apache/accumulo/pull/2674/commits/05e9e7a2b2d6bc391fc5003652cdec61132545bb)